staging/src/k8s.io/kubectl/pkg/cmd/testing/util.go

```diff
@@ -67,6 +67,50 @@ func StringBody(body string) io.ReadCloser {
 	return io.NopCloser(bytes.NewReader([]byte(body)))
 }
 
+func GetTestPod() *corev1.Pod {
```

Comment:
I think we can move this function and GetTestService into the completion_test.go. Is there any particular reason of these functions to be defined in here?

Reply:
I thought it could be used for future tests. I'll move it anyways